src/main/java/org/motechproject/mots/domain/Course.java

```diff
@@ -25,6 +25,9 @@
 import org.motechproject.mots.exception.MotsException;
 import org.motechproject.mots.validate.CourseReleaseCheck;
 
+/**
+ * This class represents higher level entity that contains {@link Module}s.
```

Comment:
The Course, Modules and Units are just used to organize the content and divide it in smaller parts to simplify progress tracking.